syntax = "proto3";

package bayesmix;

/*
 * Enum for the different types of Mixing.
 */
enum MixingId {
    UNKNOWN_MIXING = 0;
    DP = 1; // Dirichlet Process
    PY = 2; // Pitman-Yor Process
    LogSB = 3; // Logit Stick-Breaking Process
    TruncSB = 4; // Truncated Stick-Breaking Process
    MFM = 5; // Mixture of finite mixtures
    PythonMix = 6; // Generic python mixing
}
